Improve Generic x86-64 installation instructions (#26103)
home-assistant/home-assistant.github.io: source/_includes/installation/operating_system.md

1. Disable Secure Boot.
![Disable Secure Boot mode](/images/installation/intel-nuc-disable-secure-boot.jpg)

1. Save your changes and exit.

The BIOS configuration is now complete.

Next we need to write the Home Assistant Operating System image to the "boot medium", which is the medium your x86-64 hardware will boot from when it is running Home Assistant.

<div class='note'>

> HAOS has no integrated installer that writes the image automatically, you must write it manually using e.g. Etcher.

Reviewer: ```suggestion
HAOS has no integrated installer that writes the image automatically, you must write it manually using e.g. balenaEtcher.
```

Author: Although that is a correct term, it is also referred to as `Etcher` on their own pages, and in most docs in this repo.

Reviewer: You are right. I mainly saw balenaEtcher, but I only looked at one of their articles.

</div>

Typically an internal medium like S-ATA hard disk, S-ATA SSD, M.2 SSD, or a non-removable eMMC is used for the x86-64 boot medium. Alternatively an external medium can be used such as a USB SDD, though this is not recommended.

To write the HAOS image to the boot medium on your x86-64 hardware, there are 2 different methods:

1. Write the HAOS disk image from your desktop computer directly to the boot medium (e.g. using a USB to S-ATA adapter).

> If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal mediums or don't have the necessary adapter, try the next method instead.

Reviewer: ```suggestion
If you can use this method, proceed to "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)" and follow all steps. If you have non-removable internal media or don't have the necessary adapter, try the next method instead.
```

Author: `mediums` seems more correct here? Especially since the section in question is "write the image to your boot medium".

Reviewer: I cannot say with certainty that mediums would be incorrect in this particular case. I changed it for reasons of consistency, as media is the plural used in other places in this text. I based this on the Microsoft Style Guide entry on [medium, media](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/medium-media), and Merriam Webster [medium](https://www.merriam-webster.com/dictionary/medium). In the context of "something (such as a magnetic disk) on which information may be stored" they use media for the plural.

1. Create a "live operating system" on a USB device running e.g. Ubuntu ([how-to guide](https://ubuntu.com/tutorials/try-ubuntu-before-you-install)). Insert it into your system and boot the live operating system. Then follow from step 2 in "[Write the image to your boot medium](#write-the-image-to-your-boot-medium)".

<div class='note'>

When installing Etcher on Ubuntu you may need to install the fuse dependency first,
to do this run the following commands in the terminal:

```bash
sudo add-apt-repository universe
sudo apt update
sudo apt install libfuse2
```

</div>

{% endif %}

### Write the image to your boot medium

> 1. Attach the Home Assistant boot medium ({{site.installation.types[page.installation_type].installation_media}}) to your computer

Reviewer: ```suggestion
1. Attach the Home Assistant boot medium ({{site.installation.types[page.installation_type].installation_media}}) to your computer.
```

Author: The other bullet points in this document don't end with a period?

Reviewer: You are right. The periods were not there before either. I think punctuation was handled a bit inconsistently in lists. I'm basing this on [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/punctuation/periods).

{% if page.installation_type == 'odroid' %}
   If you are using a [Home Assistant Blue](/blue) or ODROID N2+, you can [attach your device directly](/common-tasks/os/#flashing-an-odroid-n2).
{% endif %}

> 2. Download and start <a href="https://www.balena.io/etcher" target="_blank">Balena Etcher</a>. (You may need to run it with administrator privileges on Windows)

Reviewer: ```suggestion
2. Download and start <a href="https://www.balena.io/etcher" target="_blank">balenaEtcher</a>. (You may need to run it with administrator privileges on Windows).
```

> 3. Select "Flash from URL"

Reviewer: ```suggestion
3. Select "Flash from URL".
```

_Select and copy the URL or use the "copy" button that appear when you hover it._

> 1. Paste the URL for the {{site.installation.types[page.installation_type].board}} image into Balena Etcher and click "OK"

Reviewer: ```suggestion
1. Paste the URL for the {{site.installation.types[page.installation_type].board}} image into balenaEtcher and click "OK".
```

![Screenshot of the Etcher software showing the URL bar with a URL pasted in.](/images/installation/etcher2.png)

> 1. When Balena Etcher has downloaded the image, click "Select target"

Reviewer: ```suggestion
1. When balenaEtcher has downloaded the image, click "Select target".
```

![Screenshot of the Etcher software showing the select target button highlighted.](/images/installation/etcher3.png)

> 1. Select the boot medium ({{site.installation.types[page.installation_type].installation_media}}) you want to use for your installation

Reviewer: ```suggestion
1. Select the boot medium ({{site.installation.types[page.installation_type].installation_media}}) you want to use for your installation.
```

![Screenshot of the Etcher software showing teh targets available.](/images/installation/etcher4.png)

> 1. Click on "Flash!" to start writing the image

Reviewer: ```suggestion
1. Click on "Flash!" to start writing the image.
```

![Screenshot of the Etcher software showing the Flash button highlighted.](/images/installation/etcher5.png)

> 1. When Balena Etcher has finished writing the image you will see a confirmation

Reviewer: ```suggestion
1. When balenaEtcher has finished writing the image you will see a confirmation.
```

Author: These bullet points purposely do not end with a period

    4. Edit the "Settings" of the VM and go "System" then "Motherboard" and select "Enable EFI"
    5. Then go to "Network" "Adapter 1" choose "Bridged Adapter" and choose your Network adapter
    <div class="note warning">

>     Please keep in mind that the bridged adapter only functions over a hardwired ethernet connection.

Reviewer: ```suggestion
    Please keep in mind that the bridged adapter only functions over a hardwired Ethernet connection.
```
